packages/app/resources/lang/de/pages/auth/register.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+
+    'title' => 'Registrierung',
+
+    'heading' => 'Registrieren',
+
+    'buttons' => [
+
+        'login' => [
+            'before' => 'oder',
+            'label' => 'mit Konto anmelden',
+        ],
+
+        'register' => [
+            'label' => 'Registrieren',
+        ],
+
+    ],
+
+    'fields' => [
+
+        'email' => [
+            'label' => 'E-Mail-Adresse',
+        ],
+
+        'name' => [
+            'label' => 'Name',
+        ],
+
+        'password' => [
+            'label' => 'Passwort',
+            'validation_attribute' => 'password',
```

Comment:
Also, shouldn't these be capitalized, @pxlrbt?
```suggestion
            'validation_attribute' => 'passwort',
```

Reply:
Those didn't make sense to me because a `min` rule kicked in which doesn't use the attribute translation. Fixed it now